docs/markdown/android/searchguide.md

```diff
@@ -0,0 +1,97 @@
+---
+title: SearchGuide
+description: A SearchGuide appends and refines a search query. They appear under [SearchField](/android/searchfield) after user submits a search input.
+fullwidth: true
+---
+
+<ImgContainer src="https://www.pinterest-assets.com/AssetLink/6cqbx311pg433xk57bo866233s36gn13/header-searchguide-and-png.png" alt="An example of a default search guide stream."/>
+
+## Usage guidelines
+
+<TwoCol>
+<Group>
+<Do title="When to use" />
+- To refine the original search query with more accurate and targeted keywords
+
+</Group>
+<Group>
+<Dont title="When not to use" />
+- To recommend related content or descriptions. Use [Tag](web/tag) instead
+- To switch between different, yet related views. Use [Tabs](/android/tabs) instead
+- To start a new search based solely on the content of the SearchGuide
+</Group>
+</TwoCol>
+
+## Best practices
+
+- SearchGuides are serial and appear in multiples. When a query doesn’t produce enough guides of sufficient quality, they shouldn’t be shown
+- Limit the number of SearchGuides to 9 per search query. If there are less number of SearchGuides present for a give query then by default we show that number. Limiting vertical scrolling helps to enhance the user experience by reducing the effort required to navigate and locate content
+
+<TwoCol>
+  <Group>
+    <ImgContainer src="https://www.pinterest-assets.com/AssetLink/p46o0t3c13c602jmda1218lv28sys6x7/do-1-and-png.png" alt="example of Search Guides placed correctly."/>
+    <Do title="Do" />
+    Place SearchGuides at the start of the screen, and scroll horizontally to reveal additional guides. **Note:** be aware that off-screen guides have significantly lower engagement.
+  </Group>
+  <Group>
+    <ImgContainer src="https://www.pinterest-assets.com/AssetLink/m78qu55yf6c557j47y26047u730nq40x/dont-1-and-png.png" alt="example of Search Guides being truncated."/>
+    <Dont title="Don't" />
+    Truncate search queries to fit within a viewport.
+  </Group>
+</TwoCol>
+
+## Accessibility
+
+- Ensure that SearchGuide component uses the correct markup for lists. Typically screen readers announce the number of items in a list and announce each item.
```

Comment:
grammar: "Ensure that _each_ SearchGuide uses the correct markup…" ("SearchGuide component" is redundant)